### PR TITLE
Add browser example

### PR DESCRIPTION
**Pull request by ThibaudDauce**

### CODE REVIEW

File: examples/browser/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    { name: "R&D", isHybridized: false },
+                    { name: "HR", isHybridized: false },
+                    ],
+                    false,
+                ),
+            ]);
+
+            const masterKeys = generateMasterKeys(policy);
+
+            const highSecretAllDepartementsDecryptionKey = generateUserSecretKey(masterKeys.secretKey, "Security Level::High && Department::R&D && Department::HR", policy)
```
Comment (Manuthor) [inline examples/browser/index.html:42]:
```suggestion
            const highSecretAllDepartmentsDecryptionKey = generateUserSecretKey(masterKeys.secretKey, "Security Level::High && Department::R&D && Department::HR", policy)
```

File: examples/browser/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            const lowSecretHRDepartmentDecryptionKey = generateUserSecretKey(masterKeys.secretKey, "Security Level::Low && Department::HR", policy)
+
+            const secret = encrypt(policy, masterKeys.publicKey, "Security Level::High && Department::HR", new TextEncoder().encode("My secret!"));
+            const decrypted = decrypt(highSecretAllDepartementsDecryptionKey, secret);
```
Comment (Manuthor) [inline examples/browser/index.html:46]:
```suggestion
            const decrypted = decrypt(highSecretAllDepartmentsDecryptionKey, secret);
```

File: examples/browser/index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    "Department",
+                    [
+                    { name: "R&D", isHybridized: false },
+                    { name: "HR", isHybridized: false },
+                    ],
+                    false,
+                ),
+            ]);
+
+            const masterKeys = generateMasterKeys(policy);
```
Comment (Manuthor) [inline examples/browser/index.html:40]: we should `console.log` something to show we are doing something :)

**Review by Manuthor — APPROVED** [review]

File: examples/browser/index.html (hunk 3 above)
Comment (ThibaudDauce) [reply examples/browser/index.html:40]: Why?

Comment (ThibaudDauce) [reply examples/browser/index.html:40]: Done in 3e8c58d8157b0cb5e9e0469d1c14a64ef0d26530

**Pull request merged**